webhook.py

```diff
@@ -567,41 +567,21 @@ def check_for_forthcoming_pushes_in_queue(submitted_json_payload:Dict[str,Any],
 
 # user_projects_invoked_string = 'user-projects.invoked.unknown--unknown'
 project_types_invoked_string = f'{job_handler_stats_prefix}.types.invoked.unknown'
-def handle_page_build(base_temp_dir_name:str, submitted_json_payload:Dict[str,Any], redis_connection,
-                        commit_type:str, commit_id:str, commit_hash:Optional[str],
-                        repo_data_url:str, repo_owner_username:str, repo_name:str,
-                        source_url_base:str, our_identifier:str,
-                        our_queue) -> str:
+def handle_catalog_build(base_temp_dir_name:str, submitted_json_payload:Dict[str, Any], redis_connection,
```

Comment:
@richmahn ok I think this is were most of my code will live then right?
If I've renamed everything properly, you should be able to enqueue a new `Door43_catalog_webhook` job which will get picked up by this handler right?

Reply:
Yes, that's correct (sorry just now seeing this, so very late, but for the record)